src/modules/AdminModule.cpp

```diff
@@ -598,6 +602,18 @@ void AdminModule::saveChanges(int saveWhat, bool shouldReboot)
     }
 }
 
+void AdminModule::handleSetHamMode(const meshtastic_HamParameters &p)
+{
+    strncpy(owner.long_name, p.call_sign, sizeof(owner.long_name));
+    owner.is_licensed = true;
+    config.lora.override_duty_cycle = true;
+    config.lora.tx_power = p.tx_power;
+    config.lora.override_frequency = p.frequency;
+
```

Comment:
I assume the unset fields will result in unset config settings? e.g. if i don't want to specify a frequency or tx_power.